app/application/use_cases/send_unlock_command.py

```
@@ -1,87 +1,146 @@
-            VaultOfflineError: If vault is not connected
+            ValueError: If vault not found.
+            UnauthorizedVaultAccessError: If user has no vault access.
+            InsufficientPermissionsError: If user's role cannot unlock.
+            VaultOfflineError: If vault is not connected.
         """
-
-        # 1. Verify vault exists
+        # Step 1: Verify vault exists
         vault = self._repo.get_by_id(vault_id)
         if vault is None:
             raise ValueError(f"Vault {vault_id} not found")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)